src/globus_sdk/scopes/_normalize.py

```diff
@@ -11,18 +11,68 @@
 
 def scopes_to_str(scopes: ScopeCollectionType) -> str:
     """
-    Convert a scope collection to a space-separated string.
+    Utility function to normalize a scope collection to a space-separated scope string.
```

Comment:
I think that the "Utility function" and "Convenience function" pieces here can be replaced with simply what the function does. The original text matches the descriptive "do a thing" phrasing that most strongly informs what the function does.

```suggestion
    Convert a scope collection to a space-separated string.
```

Reply:
Removed utility & convenience references